community/modules/scripts/spack-install/README.md

```diff
@@ -222,12 +222,12 @@ limitations under the License.
 | <a name="input_data_files"></a> [data\_files](#input\_data\_files) | A list of files to be transferred prior to running commands. <br>It must specify one of 'source' (absolute local file path) or 'content' (string).<br>It must specify a 'destination' with absolute path where file should be placed. | `list(map(string))` | `[]` | no |
 | <a name="input_deployment_name"></a> [deployment\_name](#input\_deployment\_name) | Name of deployment, used to name bucket containing startup script. | `string` | n/a | yes |
-| <a name="input_environments"></a> [environments](#input\_environments) | DEPRECATED<br><br>The following `commands` can be used to configure an environment:<pre>if ! spack env list | grep -q my-env; then<br>  spack env create my-env<br>fi<br>spack env activate my-env<br>spack add intel-mpi@2018.4.274 %gcc@10.3.0<br>spack concretize<br>spack install</pre>Defines spack environments to configure.<br>For more information, see: https://spack.readthedocs.io/en/latest/environments.html. | `any` | `null` | no |
+| <a name="input_environments"></a> [environments](#input\_environments) | DEPRECATED<br><br>The following `commands` can be used to configure an environment:<pre>if ! spack env list \| grep -q my-env; then<br>  spack env create my-env<br>fi<br>spack env activate my-env<br>spack add intel-mpi@2018.4.274 %gcc@10.3.0<br>spack concretize<br>spack install</pre>Defines spack environments to configure.<br>For more information, see: https://spack.readthedocs.io/en/latest/environments.html. | `any` | `null` | no |
 | <a name="input_gpg_keys"></a> [gpg\_keys](#input\_gpg\_keys) | GPG Keys to trust within spack.<br>  Each key must define a type. Valid types are 'file' and 'new'.<br>  Keys of type 'file' must define a path to the key that<br>  should be trusted.<br>  Keys of type 'new' must define a 'name' and 'email' to create<br>  the key with. | `list(map(any))` | `[]` | no |
 | <a name="input_install_dir"></a> [install\_dir](#input\_install\_dir) | Directory to install spack into. | `string` | `"/sw/spack"` | no |
 | <a name="input_install_flags"></a> [install\_flags](#input\_install\_flags) | Defines the flags to pass into `spack install` | `string` | `""` | no |
 | <a name="input_labels"></a> [labels](#input\_labels) | Key-value pairs of labels to be added to created resources. | `map(string)` | n/a | yes |
-| <a name="input_licenses"></a> [licenses](#input\_licenses) | List of software licenses to install within spack. | <pre>list(object({<br>    source = string<br>    dest   = string<br>  }))</pre> | `null` | no |
+| <a name="input_licenses"></a> [licenses](#input\_licenses) | DEPRECATED<br><br>The following `commands` can be used to install licenses:<pre>gsutil cp \<source\> \<dest\></pre>List of software licenses to install within spack. | <pre>list(object({<br>    source = string<br>    dest   = string<br>  }))</pre> | `null` | no |
```

Comment:
I think this is fine for replicated the behavior of the old module, but I might suggest we actually change this to using data files instead of commands.

Given the data file runners happen after setup, they should be able to provide an absolute path to spack's license directory.

Reply:
Yeah, exactly.

Reply:
Done.